spaceflights/README.md

```diff
@@ -1,16 +1,16 @@
-# Kedro starter for the Spaceflights tutorial (`kedro-starter-spaceflights`)
+# The `spaceflights` Kedro starter
 
 ## Overview
 
-This is a tutorial project, which was generated using `Kedro 0.16.5`. It is the completed version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/01_spaceflights_tutorial.html) described in the [online Kedro documentation](https://kedro.readthedocs.io) and includes the data necessary to run the project.
+This is a tutorial project, which was generated using `Kedro 0.16.5`. It is the completed version of the [spaceflights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/01_spaceflights_tutorial.html) described in the [online Kedro documentation](https://kedro.readthedocs.io) and includes the data necessary to run the project.
```

Comment:
Is this project really generated with Kedro `0.16.5`? The starters should just pick up whatever version of Kedro you're using, so this sentence is quite confusing. We should probably just delete it.

Reply:
Very good point. Done in a6908c2